mock/soap/build.gradle

```diff
@@ -24,6 +24,10 @@ dependencies {
     testImplementation project(':test:sample-soap-client')
 }
 
+test {
```

Comment:
Was this added to avoid synchronisation issues with the `ThreadLocal` model? If so, how would you feel about removing this now @driesva?

Reply:
It was rather for the following in _SoapPluginImpl_
```kotlin
init {
    // https://cwiki.apache.org/confluence/display/XMLBEANS/ExternalEntityResolver
    System.setProperty("xmlbean.entityResolver", WsdlRelativeXsdEntityResolver::class.qualifiedName!!)
}
```

If the tests (always) pass without having to set it, perfect.